pyproject.toml

```diff
@@ -34,6 +33,12 @@
         ]
         ignore_missing_imports = true
 
+    [[tool.mypy.overrides]]
+        module = [
+            "falcon.stream"
+        ]
+        disallow_untyped_defs = true
```

Comment:
what untyped defs are left?

Reply:
well none in `stream.py`, it's a config value for mypy to be strict about it.

Reply:
sorry, I read `allow..`. ok then!